build/scripts/build-impl.xml

```diff
@@ -344,11 +343,26 @@
         <echo message="Creating exist.jar ..."/>
         <jar basedir="${build.classes}" compress="true" jarfile="exist.jar">
             <manifest>
-                <attribute name="Project-Name" value="${project.name}"/>
-                <attribute name="Project-Version" value="${project.version}"/>
-                <attribute name="Project-Build" value="${DSTAMP}"/>
-                <attribute name="Project-CodeName" value="${project.codename}"/>
-                <attribute name="Git-commit" value="${git.commit}"/>
+                <attribute name="Implementation-Title" value="${project.name} Core"/>
+                <attribute name="Implementation-Version" value="${project.version}"/>
+                <attribute name="Implementation-Vendor" value="The eXist Project"/>
```

Comment:
The naming issue was raised by @joewiz some time ago ("db or not db: The many names of eXist") and agreement was to use `eXist-db`, right?